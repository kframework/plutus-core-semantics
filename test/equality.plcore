(module Foo (imported) (exported () ()) (local () ())
  (define r1 (builtin equalsInt 2 2))
  (define r2 (builtin equalsInt 2 3))
  (define r5 (builtin equalsByteString #AAA #AAA))
  (define r6 (builtin equalsByteString #AAA #111))
)
