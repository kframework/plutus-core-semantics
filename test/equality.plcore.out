<T> <k> . </k> <lenv> .Map </lenv> <genv> ( Foo . r5 ) |-> ( con Prelude . True .ValList ) ( Foo . r2 ) |-> ( con Prelude . False .ValList ) ( Foo . r1 ) |-> ( con Prelude . True .ValList ) ( Foo . r6 ) |-> ( con Prelude . False .ValList ) </genv> <mod> Foo </mod> <sym> .Map </sym> <gas> 9978 </gas> </T>
